src/events/events.ts

```diff
@@ -274,8 +275,8 @@ export async function sendEventBridgeEvent(
     // Don't halt program, but capture the failure in Sentry and Cloudwatch
     Sentry.addBreadcrumb(failedEventError);
     Sentry.captureException(error);
-    console.log(failedEventError);
-    console.log(error);
+    serverLogger.error(failedEventError);
+    serverLogger.error(error);
```

Comment:
it would be nice to do a little cleanup while we're here to make investigating errors/navigating logs easier.

for example, we could have a consistent error message with additional context, such as:

```
serverLogger.error('Failed to send event to event bus', {
    eventType: payload.eventType,
    payload: JSON.stringify(payload),
    error,
});
```

Reply:
for sure, let me make this prettier